apps/src/templates/rubrics/RubricContainer.jsx

```diff
@@ -42,6 +47,10 @@ export default function RubricContainer({
   open,
   closeRubric,
   sectionId,
+  allTeacherEvaluationData,
+  hasTeacherFeedbackMap,
```

Comment:
These two items aren't actually used in this component except to be passed down the tree to a different component. Perhaps better to connect those components to the redux directly?